cmd/suggestion/chocolate/v1alpha3/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM python:3.6
 
-RUN if [ "$(uname -m)" = "ppc64le" ]; then \
+RUN if [ "$(uname -m)" = "ppc64le" ] || [ "$(uname -m)" = "aarch64" ]; then \
         apt-get -y update && \
         apt-get -y install gfortran libopenblas-dev liblapack-dev && \
-        pip install cython; \
+        pip install cython 'numpy>=1.13.3'; \
```

Comment:
/cc @hougangliu @hmtai

Reply:
FYI: The reason why I added the pre-installation of numpy is that without this, when `pip install -r requirements.txt`, an error would occur on my arm64 machine saying that numpy should be installed first.

Reply:
Gotcha. Require @hougangliu 's review since I am not sure if it affects PPC.